htdocs/js/components/StaticDataTable.js

```diff
@@ -10,7 +12,7 @@ StaticDataTable = React.createClass({
         // parameters of the form: func(ColumnName, CellData, EntireRowData)
         getFormattedCell: React.PropTypes.func
     },
-    componentDidMount: function () {
+    componentDidMount: function componentDidMount() {
```

Comment:
Thanks for the link. I'm not a huge fan of the duplication, but if it's ES6, there's not much we can do of it.